client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -505,13 +509,12 @@
         // if there is no expiry date, this NR is not approved-expired
         if (this.$store.getters.expiryDate == null) return false
 
-        let expired_date = new Date(this.$store.state.expiryDate)
-        let date = new moment().format('YYYY-MM-DD')
-        if (this.$store.getters.currentState === "APPROVED" && date > expired_date) return true
+        if (this.$store.getters.currentState === 'EXPIRED') return true
+
         return false
```

Comment:
Same comment as above: confirm conditions and update/add a comment please.

Reply:
Comments added